.gitignore

```diff
@@ -55,4 +55,5 @@ frontend/tmp
 frontend/gis/cartodb/cartodb-config.json
 
 frontend/scripts/coverage
-
+frontend/docs/dist
+package-lock.json
```

Comment:
instead of ignoring `package-lock.json`, let's just remove it.

Reply:
I vote for ignoring it, in case we run npm and not yarn sometime, and forget to delete it

Reply:
doing that will cause bugs on your local, pushing it to github let's someone else know your living dangerously